test/data/swos-123-ref-object-reference.yaml

```diff
@@ -0,0 +1,83 @@
+openapi: 3.0.0
```

Comment:
As I mentioned in one the previous comment, we don't need such a big and complicated fixture to test on.

Reply:
fixture simplified and file renamed. if you have further comments, add to the new file. :)